util/perl/OpenSSL/Test.pm

```diff
@@ -399,7 +400,7 @@ The options that C<run> can take are in the form of hash values:
 =item B<capture =E<gt> 0|1>
 
 If true, the command will be executed with a perl backtick, and C<run> will
-return the resulting output as an array of lines.  If false or not given,
+return the resulting STDOUT output as an array of lines.  If false or not given,
```

Comment:
You still have STDOUT here.

Reply:
Ah, there was a second occurrence. Changed also.